core/perf_test/CMakeLists.txt

```diff
@@ -16,9 +16,11 @@ if(KOKKOS_ENABLE_TESTS)
 
   kokkos_include_directories(REQUIRED_DURING_INSTALLATION_TESTING ${CMAKE_CURRENT_SOURCE_DIR})
 
-  if(NOT Kokkos_ENABLE_OPENMPTARGET)
-    # FIXME OPENMPTARGET needs tasking
-    kokkos_add_executable_and_test(PerformanceTest_TaskDag SOURCES test_taskdag.cpp CATEGORIES PERFORMANCE)
+  if(Kokkos_ENABLE_DEPRECATED_CODE_4)
+    if(NOT Kokkos_ENABLE_OPENMPTARGET)
+      # FIXME OPENMPTARGET needs tasking
```

Comment:
I suspect we don't need these guards
https://github.com/kokkos/kokkos/blob/1df216f63db580a2d8172f44c05a440be982165a/core/perf_test/test_taskdag.cpp#L21-L25

Reply:
Ping

Reply:
But it makes sense to guard in `CMakelists.txt` rather than relying in `test_taskdag.cpp` to work without tasking support. It's slightly easier to find when removing deprecations as well.

Reply:
It is about not unnecessarily special casing and I don't buy that it helps with the realizing that test must go.
Who is not going to think to grep for "Task"?

Reply:
Here you go.